test/integration/scheduler/serving/healthcheck_test.go

```diff
@@ -31,9 +31,14 @@ import (
 	kubeapiservertesting "k8s.io/kubernetes/cmd/kube-apiserver/app/testing"
 	kubeschedulertesting "k8s.io/kubernetes/cmd/kube-scheduler/app/testing"
 	"k8s.io/kubernetes/test/integration/framework"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	featuregatetesting "k8s.io/component-base/featuregate/testing"
+	zpagesfeatures "k8s.io/component-base/zpages/features"
 )
 
 func TestHealthEndpoints(t *testing.T) {
```

Comment:
Similarly here, can you rename it to TestEndpointHandlers and the file itself?

Reply:
Done.